docs/private-apis.md

```diff
@@ -0,0 +1,348 @@
+# Gutenberg Private APIs
+
+Gutenberg packages currently export 237 private APIs:
+- 42 store actions
+- 57 store selectors
+- 66 React components
+- 30 React hooks
```

Comment:
Do the numbers have any specific utility? I'd refrain from adding those numbers because it will be very easy for them to get obsolete.

Reply:
I don't understand this feedback. Is is about the usefulness of the numbers (I think they are very interesting!) or about them being placed in the document, rather than is some (interesting) GitHub comment?